lib/phoenix_test/case.ex

```diff
@@ -27,14 +27,17 @@ defmodule PhoenixTest.Case do
   ]
 
   setup_all context do
-    trace = Map.get(context, :trace, Application.fetch_env!(:phoenix_test, :playwright)[:trace])
+    global_config = Application.fetch_env!(:phoenix_test, :playwright)
+    global_browser_config = List.wrap(global_config[:browser])
+    trace = Map.get(context, :trace, global_config[:trace])
 
     case context do
       %{playwright: true} ->
-        [browser_id: Case.Playwright.launch_browser(@playwright_opts), trace: trace]
+        opts = Keyword.merge(@playwright_opts, global_browser_config)
+        [browser_id: Case.Playwright.launch_browser(opts), trace: trace]
 
       %{playwright: opts} when is_list(opts) ->
-        opts = Keyword.merge(@playwright_opts, opts)
+        opts = Keyword.merge(@playwright_opts, global_browser_config)
```

Comment:
This looses the "local" `opts` passed via ExUnit `@[module,describe]tag playwright: ...`, doesn't it?

Reply:
Doh, right you are! Fixed in afb37856d1a095a2c9a210deaa43098328904cce and tested locally.